src/argilla/client/feedback/utils/html.py

```diff
@@ -0,0 +1,210 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+
+import base64
+from pathlib import Path
+from typing import Callable, List, Optional, Union
+
+import matplotlib.pyplot as plt
+from matplotlib.colors import rgb2hex
+
+# Define supported types for each media
+SUPPORTED_MEDIA_TYPES = {
+    "video": ["mp4", "webm"],  # .ogg does not work in Safari
+    "audio": ["mp3", "wav"],  # .ogg does not work in Safari
+    "image": ["png", "jpg", "gif"],  # basic one, but more can be added
+}
+
+
+def media_to_html(media_type: str, path: str, file_type: Optional[str] = None) -> str:
```

Comment:
Hi, could we also potentially allow for passing a byte string directly instead of passing a path? Perhaps we can check to allow for a path or non-b64encoded string.